### PR TITLE
[Components with more than 1 service] LPS-179094 Refactor ScreenNavigationCategory components from Objects modules

### PR DESCRIPTION
**Pull request by RenanUmV**
[LPS-179094](https://issues.liferay.com/browse/LPS-179094)

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:object

**Comment by Kvale1733** [comment]
ci:test:stable

**Comment by Kvale1733** [comment]
@RenanUmV No regressions.

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:stable

**Comment by RenanUmV** [comment]
ci:test:object

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by RenanUmV** [comment]
ci:test:object

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by RenanUmV** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/frontend/taglib/servlet/taglib/ActionsObjectDefinitionsScreenNavigationEntry.java [hunk 1]
```diff
@@ -61,20 +53,14 @@ public String getCategoryKey() {
 
 	@Override
 	public String getEntryKey() {
-		return ObjectDefinitionsScreenNavigationEntryConstants.
-			ENTRY_KEY_ACTIONS;
+		return getCategoryKey();
```
Comment (dantewang) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/frontend/taglib/servlet/taglib/ActionsObjectDefinitionsScreenNavigationEntry.java:56]: @RenanUmV Could you also extract this method to the base class, as all the sub classes of `BaseObjectDefinitionScreenNavigationEntry` have this method delegating to `getCategoryKey()`.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by RenanUmV** [comment]
ci:test:object

**Comment by zhangyan0701** [comment]
No regressions found.

**Pull request closed without merging**